packages/statusLambda/src/statusLambda.ts

```diff
@@ -31,11 +31,28 @@ const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPro
 
   const spineClient = createSpineClient()
 
-  const spineStatus = await spineClient.getStatus(logger)
-
   const commitId = process.env.COMMIT_ID
   const versionNumber = process.env.VERSION_NUMBER
 
+  const isCertificateConfigured = await spineClient.isCertificateConfigured()
```

Comment:
no need for await

Reply:
The code has been amended - async has been removed